.gitignore

```diff
@@ -14,3 +14,4 @@ TAGS
 /ligo/*.tz
 # We also ignore them in `haskell/test/resources/` because they are symlinked for tests.
 /haskell/test/resources/*.tz
+!/haskell/test/resources/transferlist.tz
```

Comment:
Since we decided to support only monolithic contract, it might be simpler to have ` /haskell/test/resources/stablecoin.tz` here instead of 2 lines.